lib/jnpr/junos/device.py

```diff
@@ -833,11 +833,11 @@ def __init__(self, *vargs, **kvargs):
             self._conf_ssh_private_key_file = None
             # user can get updated by ssh_config
             self._ssh_config = kvargs.get('ssh_config')
+            self._sshconf_lkup()
```

Comment:
I agree adding this line which will override self._conf_ssh_private_key_file None value set in line 833 if that is configured in ssh config file.